src/modules/auth/login.controller.ts

```diff
@@ -42,6 +42,9 @@ export class LoginController {
   @Post('login')
   async login(@Req() req: Request, @Res() res: Response) {
     const origin = req.headers['origin'];
+    if (origin.includes('switchboard')) {
+      throw new Error('Switchboard is not allowed!');
+    }
```

Comment:
The objective is to trigger the health-checks to fail for ssi-hub, not break from a functional stand point (which is what I think this is).